DuckDuckGo/FavoritesHomeViewSectionRenderer.swift

```diff
@@ -89,20 +89,12 @@ class FavoritesHomeViewSectionRenderer {
     }
 
     func sectionMargin(in collectionView: UICollectionView) -> CGFloat {
-        if controller is FavoritesOverlay {
-            return Constants.largeModeMargin
-        }
-        
-        let margin: CGFloat
         if isPad {
```

Comment:
I'm afraid this condition is insufficient and we also need to check for a size class in case we're in a Split View:
<img src="https://github.com/user-attachments/assets/a9456be8-cede-41b1-b32e-9dfe0d5ecb40" width=350 />
The same goes for favorites in Bookmarks:
<img src="https://github.com/user-attachments/assets/b1b45002-2483-4b68-973b-c1af34d7da2c" width=350 />